src/components/UriViewer.js

```diff
@@ -122,7 +120,7 @@ const DownloadButton = ({ uri, metadata: { bucket, name, fileName, size }, acces
         const { url } = await Ajax(signal).Martha.getSignedUrl({
           bucket,
           object: name,
-          dataObjectUri: isDrs(uri) && uri
+          dataObjectUri: isDrs(uri) ? uri : null
```

Comment:
I think we want undefined here, right? So that the field is just left off? Is there a reason it should be defined and null?

Reply:
We use [this rule](https://eslint.org/docs/rules/no-shadow-restricted-names#disallow-shadowing-of-restricted-names-no-shadow-restricted-names) to make sure that you get `undefined` when you write `undefined`, so I'd prefer that you use `undefined` here, unless the server requires that the field be present but null.